src/neo/Network/P2P/Payloads/PluginPayload.cs

```diff
@@ -0,0 +1,110 @@
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.SmartContract.Native.Designate;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class PluginPayload : IVerifiable
+    {
+        public string Plugin;
+        public byte MessageType;
```

Comment:
And how about defining an enum for this variable?

Reply:
It depends to the plugin

Reply:
Then I think `MessageType` should be included in `Data`.

Reply:
Done